src/pybind/mgr/nfs/export.py

```diff
@@ -425,20 +426,21 @@ def list_all_exports(self) -> List[Dict[str, Any]]:
     @export_cluster_checker
     def list_exports(self,
                      cluster_id: str,
-                     detailed: bool = False) -> Tuple[int, str, str]:
+                     detailed: bool = False) -> List[Any]:
         try:
             if detailed:
                 result_d = [export.to_dict() for export in self.exports[cluster_id]]
-                return 0, json.dumps(result_d, indent=2), ''
+                return result_d
```

Comment:
so to get it clear, the Responder decorator handles the conversion from whatever return type we have here to the `Tuple[int, str, str]` we always had to generate ourselves before?

Reply:
Can you please add to the PR description some details about how this helper can infer the return code or the stderr output? Thanks!